DigitalLearningSolutions.Web/Views/Supervisor/Shared/_TagRow.cshtml

```diff
@@ -0,0 +1,14 @@
+﻿@using DigitalLearningSolutions.Data.Models.SelfAssessments;
+
+@model bool;
+
+
+<div class="tags">
+        @if (Model)
```

Comment:
don't you think that this condition is better suited in the main view rather than the partial? I mean if the competency is not optional, then we won't have to come to this partial.

Reply:
The reason why I did not put condition on the main page is that I was thinking if we want to add additional tag to this _TagRow.cshtml in the future it make it simple and easy doing that